src/model/receipt.ts

```diff
@@ -0,0 +1,42 @@
+import { Address, Id, Receipt, ReceiptType, ReceiptVersion} from 'nem2-sdk';
+
```

Comment:
Do not create your own receipt class. Use the ones provided by the SDK, and separate by type with if's:

* [ArtifactExpiry](https://nemtech.github.io/nem2-sdk-typescript-javascript/modules/_model_receipt_artifactexpiryreceipt_.html ): Type (hexa) starts with "0x4".
* [BalanceTransfer](https://nemtech.github.io/nem2-sdk-typescript-javascript/modules/_model_receipt_balancechangereceipt_.html) : Type (hexa) starts with "0x2" (BalanceCredit) or "0x3" (BalanceDebit).
* [BalanceTransfer](https://nemtech.github.io/nem2-sdk-typescript-javascript/modules/_model_receipt_balancetransferreceipt_.html): Type (hexa) starts with "0x1"
* [InflationReceipt](https://nemtech.github.io/nem2-sdk-typescript-javascript/modules/_model_receipt_inflationreceipt_.html): Type (hexa) starts with "0x5".